### ISSUE TITLE
Show duplicates has a bug

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#289]
1. Please check **Show duplicates** > Bathrooms > 0 
it should show cases with Bathrooms ==0 and the number of records should be 119 rows. now it shows unrelated records (http://alphatechadmin.pythonanywhere.com/dtale/main/7)

2. **Show duplicates and execute has bug**:
       **Show duplicates** > group by **exterior color** > **execute** dtale replace the **exterior color** with Null and shows an error. may be because I didn't select any case. in my opinion when any case aren't selected the Execute button should be inactive
![image](https://user-images.githubusercontent.com/2202483/96253022-de048200-0f80-11eb-85fb-ecf2bcbffae5.png)

**Comment by aschonfeld** [comment]
1. seems to work for me no problem
2. The execute button should still work even without a selection.  That just means you want to filter all duplicates and not just specific groups.  This worked fine for me.  I can add display of the total number of duplicates at the top of all the groups though.

**Comment by reza1615** [comment]
https://youtu.be/OQ8pHI8oHU0

**Comment by aschonfeld** [comment]
So I did some testing locally and everything seemed to work fine.  I found a bug with selecting a specific group of duplicates when the column you're grouping on has spaces in its name. (this will be included in the next release)

I was able to see the error you had on the demo site, but once again I'm going to have to chalk it up to the fact that the demo site is hosted in an odd way where I believe pythonanywhere spins up multiple instances of the app and so each request isn't necessarily the same instance you were working off and create some odd behavior.  If you're able to see this behavior when using the tool locally then I'd be more worried.
